src/Mapbender/CoreBundle/Resources/public/mapbender.element.searchRouter.js

```diff
@@ -398,37 +395,56 @@
         /**
          * Rebuilds result table with search result data.
          *
-         * @param SearchModel       model   Search model
-         * @param FeatureCollection results Search result feature collection
-         * @param object            options Backbone options
+         * @param model             SearchModel
+         * @param results           FeatureCollection
+         * @param options           object Backbone options
          */
```

Comment:
Type documentation uses curly brackets, fix it or leave it alone.
http://usejsdoc.org/tags-param.html